BLSQ/openhexa-frontend
# feat(Workspaces): rename and add check on exit preview button

Rename and add check on exit preview button.

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Rename Exit preview to 'Back to OpenHexa classic'
- Hide button if user doesn't have openhexa_legacy feature flag.

## How/what to test

- Create a user with openhexa_legacy enabled and check if the button is displayed. 
- Create a user with only workspaces feature flag enable and check that the button is hidden.

## Screenshots / screencast

User with openhexa_legacy feature flag
![image](https://user-images.githubusercontent.com/25453621/235978228-c02b0c71-e4c9-4735-826d-4266d056e75a.png)

User with only workspaces feature flag
![image](https://user-images.githubusercontent.com/25453621/235978364-fb8e7931-c05b-4ad4-988b-9dbde211c810.png)
